### PR TITLE
Yulian47_stop-watch

### PR DESCRIPTION
**Pull request by Yulian47**
- [DEMO LINK](https://<Yulian47>.github.io/layout_stop-watch/)

❗️ Copy this `Checklist` to the `Pull Request` description after links, and put `- [x]` before each point after you checked it.

- [x] Keyframes implemented using from/to + transform with rotate property
- [x] Stopwatch is centered and has correct arrows size
- [x] All `Typical Mistakes` from `BEM` lesson theory are checked.
- [x] Code follows all the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules)

### CODE REVIEW

File: src/styles/index.scss [hunk 1]
```diff
@@ -1,3 +1,71 @@
-body {
+* {
```
Comment (DarkMistyRoom) [inline src/styles/index.scss:1]: Don't use * selector for zeroing out your margins or paddings. It's still inefficient for browser
to read your web document

Comment (DarkMistyRoom) [reply src/styles/index.scss:1]: don't apply anything except box-sizing to *

File: src/styles/index.scss [hunk 2]
```diff
@@ -1,3 +1,71 @@
...
+  left: 39.5vmin;
+
+  width: 1.5vmin;
+  height: 38vmin;
+  background-color: #2c8000;
+  animation: speed-up 10s linear infinite;
+  transform-origin: bottom left;
+}
+
+span {
```
Comment (DarkMistyRoom) [inline src/styles/index.scss:53]: better use class

**Review by DarkMistyRoom — CHANGES_REQUESTED** [review]
Fix the Demo Link

File: src/styles/index.scss (hunk 2 above)
Comment (Yulian47) [reply src/styles/index.scss]: I can't understand why  DEMO on github is empty.

**Comment by Yulian47** [comment]
> Fix the Demo Link

I can't understand why DEMO on github is empty.

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Still not fixed from the previous review, you need to add all demo link from the readme.md file and update it in the PR description

**Pull request closed without merging**